jax/experimental/jax2tf/call_tf.py

```diff
@@ -147,13 +149,34 @@ def acc_ct(res_, ct_res_):
 
 
 # The impl will be used in op-by-op mode and calls func_tf in TF eager mode.
-def _call_tf_impl(*args, args_treedef, func_tf, **_):
-  # TODO(necula): does this copy to the host?
-  args_tf_flat = [tf.constant(np.asarray(a)) for a in args]
+def _call_tf_impl(*args_jax_flat, args_treedef, func_tf, **_):
+  def _arg_jax_to_tf(arg_jax):
+    # We try to avoid making a copy of JAX arrays
+    if (isinstance(arg_jax, xla.DeviceArray) and
+        arg_jax.dtype in dlpack.SUPPORTED_DTYPES and
+        arg_jax.device_buffer.client.platform != "tpu"):
```

Comment:
I'd suggest an allow list here, I think most of our backends do not support DLPack (e.g. tpu_driver or some internal only backends).

Reply:
Good point. It turns out we only use DLPack on GPU now.